docs/api-and-integration/metadata/jira.md

```diff
@@ -0,0 +1,58 @@
+
+## Octopus Connect App and the Jira Extension
+
+Something worth pointing out at this point is that the Jira functionality we're demonstrating here is only available in the SaaS version of Jira. There isn't a SaaS requirement on the Octopus side of the integration though, so you can use either a self-hosted or Cloud version of Octopus.
+
+With that in mind, the first step in the integration is to add the Octopus Connect App in the Jira marketplace.
+
+***TODO: add a screenshot of the marketplace tile***
+
+To complete the installation of the app from the marketplace you must configure some security related values. This requires aligning a couple of pieces of information from the page that appears in Jira:
+
+![Jira ConnectApp configuration](jira-app-config.png)
+
+And from the **Configuration / Jira Issue Tracker** page in Octopus:
+
+![Octopus Jira Issue Tracker configuration](octo-jira-config.png)
+
+You'll probably want both of these open in separate browser windows/tabs to complete the configuration. On the Jira side you need to copy the _Octopus Installation Id_ from the Octopus page, and on the Octopus side you'll need to copy your Jira instance's *Base Url* and the *Octopus Secret* that appears on the Connect App configuration page to use as the *Jira Connect App Password*.
+
+## Environment Settings
+
+The next thing to configure relates back to our earlier note on environment types VS environment names. In Octopus you'll need to map the Jira environment types for the environments that you're going to be tracking work-items for:
+
+![Octopus Environment](octo-env.png)
+
+The fixed list of environment types are important for the tracking in Jira and this configuration allows you the flexibility of easily mapping any existing environments to a type.
+
+The environment name does get passed through to Jira, as shown in the earlier example, so the information is still available for you to see. Also note that if you are using multi-tenancy this is fully supported and what you will see in Jira is a per tenant per environment name. As an example, the following diagram illustrates an un-tenanted deployment to Dev and then a tenanted deployment to Dev of the same version (the text is truncated in the shot, the alt text for the link reads `Deploy work-items test release 0.0.5 to Dev for Tenant A`)
+
+![Jira Multi-Tenant progress](jira-multi-tenant.png)
+
+## Deployment Task Log
+
+Once the Jira issue tracker is enabled and configured you will see blocks similar to the following appear in the log during your deployments. These show the state updates Octopus is sending through to Jira, and if you expand them the details include information on the work-item IDs etc for traceability.
+
+The following illustrates where Octopus tried to send an _in_progress_, and then a _successful_, state change to Jira but was unable to. **This does not impact the Octopus deployment itself, it will still be considered a successful deployment.**
```

Comment:
Can we show a success one instead of a failure one?

Also, can we have these blocks only shown in verbose mode? _think this isn't currently possible - pretty sure it'd take some tweaking of the task log renderer_

Reply:
If it's important to mention that a failure in the jira stuff will not impact the success or a deployment, then this should be a log message in the code.

Also, it feels it should be logged as a warning, not an error.